examples/with-thirdparty/backend/index.ts

```diff
@@ -0,0 +1,46 @@
+import express from "express";
+import cors from "cors";
+import supertokens from "supertokens-node";
+import { verifySession } from "supertokens-node/recipe/session/framework/express";
+import { middleware, errorHandler, SessionRequest } from "supertokens-node/framework/express";
+import { SuperTokensConfig } from "./config";
+import Multitenancy from "supertokens-node/recipe/multitenancy";
+
+supertokens.init(SuperTokensConfig);
+
+const app = express();
+
+app.use(
+    cors({
+        origin: "http://localhost:3000",
+        allowedHeaders: ["content-type", ...supertokens.getAllCORSHeaders()],
+        methods: ["GET", "PUT", "POST", "DELETE"],
+        credentials: true,
+    })
+);
+
+// This exposes all the APIs from SuperTokens to the client.
+app.use(middleware());
+
+// An example API that requires session verification
+app.get("/sessioninfo", verifySession(), async (req: SessionRequest, res) => {
+    let session = req.session;
+    res.send({
+        sessionHandle: session!.getHandle(),
+        userId: session!.getUserId(),
+        accessTokenPayload: session!.getAccessTokenPayload(),
+    });
+});
+
+// This API is used by the frontend to create the tenants drop down when the app loads.
+// Depending on your UX, you can remove this API.
+app.get("/tenants", async (req, res) => {
+    let tenants = await Multitenancy.listAllTenants();
+    res.send(tenants);
+});
```

Comment:
why do we have this API here?

Reply:
This is in the backend generated by the CLI. Ill remove it though, dont need it here